src/routes/classes/[classId]/modules/[moduleId]/+page.svelte

```
@@ -385,48 +386,60 @@
-			{client}
-			{module}
-			suppressAuthErrors={true}
-		/>
+{#if data.isPublicView}
+	<!-- Public view for crawlers and unauthenticated users -->
+	<div class="flex items-center justify-center min-h-[60vh]">
+		<div class="text-center max-w-md mx-auto p-8">
+			<BookOpen class="mx-auto mb-4 text-primary" size={48} />
+			<h1 class="text-2xl font-bold mb-2">{data.seo?.title?.replace(' — LearnTerms', '') ?? 'Study Module'}</h1>
+			<p class="text-base-content/70 mb-6">Sign in to start studying this module on LearnTerms.</p>
+			<a href="/sign-in" class="btn btn-primary">Sign in to study</a>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)